docs/languages/Java.md

```diff
@@ -57,3 +57,12 @@ Check out this [example for a live demonstration](../../examples/java-generate-j
 To generate Java data models with Jackson annotation using `JAVA_JACKSON_PRESET` option.
 
 Check out this [example for a live demonstration](../../examples/java-generate-jackson-annotation).
+
+## Include JSON marshaling and unmarshaling methods
+
+Sometimes you just want to convert your class to JSON without the use of annotations such as Jackson.
+
+Check out this [example for a live demonstration](https://github.com/asyncapi/modelina/blob/master/examples/java-generate-marshal).
```

Comment:
Don't see the actual example anywhere? Please re-read https://github.com/asyncapi/modelina/pull/596#pullrequestreview-874221410